dear_petition/petition/export/documents/expungable_summary.py

```diff
@@ -97,38 +97,40 @@ def __create_tables_data(offenses):
         jurisdiction = JURISDICTION_MAP[offense.ciprs_record.jurisdiction]
         key = (county, jurisdiction)
 
-        offense_records = list(offense.offense_records.all())
-        if offense.is_convicted():
-            # remove one of the offense records because the two offense records would appear identical in the output
-            offense_records.pop()
+        offense_records = list(offense.offense_records.all().filter(is_visible=True))
```

Comment:
The unfortunate part of model properties is that you can't use them to filter like you can with the database fields. I think it's fine to replace with `[offense_record for offense_record in list(offense.offense_records.all()) if offense_record.is_visible]`.